lib/screens/v2/candidate_generator/widgets/departures.ex

```diff
@@ -54,7 +53,7 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Departures do
         Keyword.get(options, :departure_fetch_fn, &Departure.fetch/2),
         Keyword.get(options, :post_process_fn, fn results, _config -> results end),
         Keyword.get(options, :route_fetch_fn, &Route.fetch/1),
-        now
+        options[:now]
```

Comment:
It seems like this is now effectively a required argument, rather than an "option" with a default value — if you didn't provide it, downstream code would crash when it expected a `DateTime` but got `nil`. I think if it's optional it should be a keyword with a default value, but if it's required it should be its own argument. No preference as to which.

Reply:
Yeah good point, didn't think about that. Making it optional seems fine and lines up with the rest of the list.

Reply:
Actually, I think I'm changing my mind. We already default `now` in the app CGs so there's already the variable available to pass to this.